File: Java/Programs-based-on-Numbers/35-number-of-digits-in-an-integer.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter a number:");
        int num = getInput.nextInt();
        int count = 0;

        if (num == 0) {
            System.out.println("\nNumber of digits: 1");
        } else {
            while (num > 0) {
                num /= 10;
                count++;
            }
            System.out.println("\nNumber of digits: " + count);
        }
        getInput.close();
    }
}

// Using built-in method
// String num = "597564";
// int count = num.length();

File: Python/2-Programs-based-on-Numbers/35-number-of-digits-in-an-integer.py
num = int(input("Enter a number: "))
count = 0

if num == 0:
    print("\nNumber of digits: 1")
else:
    while num > 0:
        count += 1
        num = num//10

    print("\nNumber of digits:",count)


# Using built-in len() method
# num = input("Enter a number: ")
# print("Number of digits:", len(num))